bulbs/feeds/views.py

```diff
@@ -40,3 +43,52 @@ def get_context_data(self, *args, **kwargs):
             content.feed_url = self.request.build_absolute_uri(feed_path)
 
         return RequestContext(self.request, context)
+
+
+class SpecialCoverageRSSView(ContentListView):
+    """Really simply, this syndicates Content."""
+    template_name = "feeds/rss.xml"
+    paginate_by = 20
+    feed_title = "Special Coverage RSS Feed"
+    utm_params = "utm_medium=RSS&amp;utm_campaign=feeds"
+
+    def get_template_names(self):
+        return ["feeds/rss.xml", "feeds/_rss.xml"]
+
+    @cache_control(max_age=600)
+    def get(self, request, *args, **kwargs):
+        response = super(SpecialCoverageRSSView, self).get(request, *args, **kwargs)
+        response["Content-Type"] = "application/rss+xml"
+        return response
+
+    def get_queryset(self):
+        sc_id = self.request.GET.get("id")
```

Comment:
Can we prefix these parameters with something a little more descriptive?  Maybe `special_coverage_id` / `special_coverage_slug`